configuration/pih/htmlforms/hiv/section-hiv-history.xml

```diff
@@ -711,6 +721,47 @@
         </section>
     </includeIf>
 
+    <section id="contacts" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+            headerCode="pihcore.contacts">
+        <repeat with="['1'],['2'],['3'],['4'],['5']">
+            <div class="section-container mb-4 flex-questions">
+                <obsgroup groupingConceptId="PIH:Contact construct">
+                    <div>
+                        <obs conceptId="PIH:RELATIONSHIP OF RELATIVE TO PATIENT"
+                         labelCode="pihcore.contactRelationship"
+                         answerConceptIds="PIH:PARTNER OR SPOUSE,PIH:GUARDIAN RELATIONSHIP TO CHILD,PIH:SIBLING,PIH:OTHER RELATIVE"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:FIRST NAME" labelCode="pihcore.firstName"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:LAST NAME" labelCode="pihcore.lastName"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:AGE" labelCode="pihcore.age"/>
+                    </div>
+                    <div style="margin-right: 64px;">
+                        <label><uimessage code="pihcore.dead"/></label>
+                        <p class="radio">
+                            <obs conceptId="CIEL:163533"
+                                answerConceptIds="PIH:ALIVE,PIH:DEATH,PIH:UNKNOWN"
+                                answerCodes="general.no,general.yes,pihcore.unknown"
+                                style="radio"/>
+                        </p>
+                    </div>
+                    <div>
+                        <label><uimessage code="pihcore.hiv.serologicalStatus"/></label>
+                        <p class="radio">
+                            <obs conceptId="PIH:RESULT OF HIV TEST"
+                                answerConceptIds="PIH:UNKNOWN,PIH:POSITIVE,PIH:NEGATIVE"
```

Comment:
Did you mean to update this line too?  Indetermine?

Reply:
See above